src/App.tsx

```diff
@@ -15,48 +17,107 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABETIC,
+  LENGTH,
+}
+
+type OrderOptions = {
+  sortField: SortType,
+  isReversed: boolean;
+};
+
+function getPreparedGoods(
+  goods: string[],
+  { sortField, isReversed }: OrderOptions,
+) {
+  const preparedGoods = [...goods];
+
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      switch (sortField) {
+        case SortType.ALPHABETIC:
+          return good1.localeCompare(good2);
+        case SortType.LENGTH:
+          return good1.length - good2.length;
+        default:
+          return 0;
```

Comment:
It would be great to move switch...case out of the sort callback so there is no need to extra call it on default case